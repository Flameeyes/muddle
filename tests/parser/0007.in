How about ifeq?

Fish ${fn:ifeq("/top/fish", "soup")

Fish = Soup
And Fish = ${/top/fish}
}

Soup ${fn:ifeq("/top/fish", "fish")

Fish = Fish
}
