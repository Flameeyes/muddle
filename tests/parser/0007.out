How about ifeq?

Fish 

Fish = Soup
And Fish = soup


Soup 
